migrations/versions/83a6b3b09eca_add_verified_fields.py

```diff
@@ -0,0 +1,55 @@
+"""add verified fields
+
+Revision ID: 83a6b3b09eca
+Revises: 166a3402c391
+Create Date: 2024-08-28 11:14:27.902082
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "83a6b3b09eca"
+down_revision = "166a3402c391"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+
+    with op.batch_alter_table("message", schema=None) as batch_op:
+        batch_op.alter_column("content", existing_type=sa.TEXT(), nullable=True)
+
+    with op.batch_alter_table("users", schema=None) as batch_op:
+        batch_op.add_column(sa.Column("extra_field_verified1", sa.Boolean(), nullable=True))
+        batch_op.add_column(sa.Column("extra_field_verified2", sa.Boolean(), nullable=True))
+        batch_op.add_column(sa.Column("extra_field_verified3", sa.Boolean(), nullable=True))
+        batch_op.add_column(sa.Column("extra_field_verified4", sa.Boolean(), nullable=True))
+        batch_op.alter_column("password_hash", existing_type=sa.VARCHAR(length=512), nullable=False)
+        batch_op.alter_column("is_verified", existing_type=sa.BOOLEAN(), nullable=False)
```

Comment:
The auto-genereated migration includes a bunch of stuff unrelated to this PR. That's what #530 is for, to make it so these auto-generated migrations just include what's changed. Can you strip this down to only include the extra_field_verified columns on the user table and just delete the rest of the lines?

Reply:
Changes in 32b7af2